templates/shared/forms/interactive/ai.html

```diff
@@ -0,0 +1,120 @@
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+        <div class="row--50-50 u-no-padding">
+          <div class="col">
+            <h2 class="p-heading--1" id="modal-title">Open source AI: a scalable path to production</h2>
+            <p>Looking to scale your MLOPs infrastructure or need consulting services to kick start your AI journey? Our experts are here to help you.</p>
+          </div>
+          <div class="col">
+            <form action="/marketo/submit" onsubmit="stringifyCustomFields()" method="post" id="mktoForm_%% formid %%" class="modal-form">
+              <div class="p-section--shallow">
+                <p>Do you already have any AI projects rolled out in your enterprise?</p>
+                <label class="p-radio">
+                  <input type="radio" value="Yes" class="p-radio__input" id="aiProjectsYes" name="aiProjects" aria-labelledby="aiProjectsYes" required>
+                  <span class="p-radio__label" id="aiProjectsYes">Yes</span>
+                </label>
+                <label class="p-radio">
+                  <input type="radio" value="No" class="p-radio__input" id="aiProjectsNo" name="aiProjects" aria-labelledby="aiProjectsNo" required>
+                  <span class="p-radio__label" id="aiProjectsNo">No</span>
+                </label>
+              </div>
+              <div class="p-section--shallow">
+                <hr class="p-rule">
+                <p>What are you interested in from our AI offering?</p>
+                <label class="p-checkbox js-checkbox">
+                  <input type="checkbox" aria-labelledby="canonicalMlopsSolution" class="p-checkbox__input">
+                  <span class="p-checkbox__label" id="canonicalMlopsSolution">Canonical MLOps solution</span>
+                </label>
+                <label class="p-checkbox js-checkbox">
+                  <input type="checkbox" aria-labelledby="aiConsultingServices" class="p-checkbox__input">
+                  <span class="p-checkbox__label" id="aiConsultingServices">AI consultancy services</span>
+                </label>
+                <label class="p-checkbox js-checkbox">
+                  <input type="checkbox" aria-labelledby="mlopsWorkshop" class="p-checkbox__input">
+                  <span class="p-checkbox__label" id="mlopsWorkshop">MLOps Workshop</span>
+                </label>
+                <label class="p-checkbox js-checkbox">
+                  <input type="checkbox" aria-labelledby="notSure" class="p-checkbox__input">
+                  <span class="p-checkbox__label" id="notSure">Not sure</span>
+                </label>
+              </div>
+              <div class="p-section--shallow">
+                <hr class="p-rule">
+                <label for="mlopsRequirements">Add details about your AI or MLOps requirements</label>
+                <textarea id="mlopsRequirements"></textarea>
+                
```

Comment:
There is one line added by Sophie missing that says "Add your information"?

Reply:
@Sophie-32 @juanruitina @lyubomir-popov in my opinion it looks better without that line. It looks a bit out of place. See:
![Screenshot from 2023-09-11 09-48-46](https://github.com/canonical/ubuntu.com/assets/17607612/636211e9-31fa-4f05-8469-7163763a13d4)
 
vs

![image](https://github.com/canonical/ubuntu.com/assets/17607612/c5ab9777-a454-45b1-821a-aa96e1eca6e0)


If we have to keep it, what's the ask from a design perspective (it wasn't included in the mock-up) to make it look a bit better? An hr on top? Lmk what you all think is best and I'll implement ASAP!

Reply:
I'd prefer to keep it, but will keep this in mind for future forms. I defer to @lyubomir-popov on adding the HR, but that seems like the best option from my perspective.